cypress/integration/User Management/BL_00_ReceptionistEdit.js

```diff
@@ -0,0 +1,128 @@
+describe("Fills out form with mock data of receptionist from our user.json file, view our form to see if the data is inputed properly, edit our form with edited data, and submits our form", () => {
+    before(() => {
+        cy.fixture("users.json").then(({ receptionist }) => {
+            cy.mockGraphQL({
+                "CreateAdmin": {
+                    "response": {
+                        "data": {
+                            "createAdmin": {
+                                "created": false,
+                            },
+                        },
+                    },
+                    "test": (variables) => {
+                        Object.entries(variables).forEach(([key, value]) => {
+                            if (!["phoneNumber", "id", "user", "password"].includes(key)) {
+                                expect(receptionist[key] || receptionist?.user[key]).equals(value);
+                            }
+                        });
+                    },
+                },
+                "GetAdmin": {
+                    "response": {
+                        "data": {
+                            "admin": {
+                                ...receptionist,
+                                "password": null,
+                            },
+                        },
+                    },
+                    "test": ({ userID }) => {
+                        expect(userID).equals(receptionist.user.id.toString(), "Check ID passed");
+                    },
+                },
+            });
+            cy.visitAuthenticated(`/form/admin/${receptionist.user.id}`);
+        });
+    });
+
+    it("Loads our mock data properly into the login details form section for receptionist", () => {
+        cy.fixture("users.json").then(({ receptionist }) => {
+            cy.get("[data-cy=login-firstName-input]")
+                .should("have.value", receptionist.user.firstName);
+            cy.get("[data-cy=login-lastName-input]")
+                .should("have.value", receptionist.user.lastName);
+            cy.get("[data-cy=login-email-input]")
+                .should("have.value", receptionist.user.email);
+            cy.get("[data-cy=login-nextButton]")
+                .should("be.disabled");
+        });
+    });
+
+    it("Can enter data from the user properly into the first form, while checking if bad info is entered the submit button should be disabled", () => {
```

Comment:
1. Rename: "can enter new receptionist account details in the first section(??)" 
2. Separate out your tests - you're testing for 2 things: 1) entering data 2) data validation. these things shouldn't be part of the same `it()` command
3. You don't need to check for data validation since Neil already has a test for that for the form component.